docs/guide/index.md

```diff
@@ -44,7 +44,7 @@ Vite requires [Node.js](https://nodejs.org/en/) version >=12.2.0. However, some
 With NPM:
 
 ```bash
-$ npm init vite@latest
+$ npm create vite
```

Comment:
Last time I checked, `@latest` was still needed for `npm create` as it is implemented as a direct alias of `init`. Could you confirm this?

Reply:
That was correct. I fixed it.